prime-router/src/main/kotlin/cli/tests/LoadTestSimulator.kt

```diff
@@ -362,16 +381,41 @@ class Simulator : CoolTest() {
                 it.simulation.numThreads
         }
         println("Expecting $expectedResults total items. More than this may be found if other test")
-        // if we are running in async mode, verify the correct number of 'process' records have been generated
-        if (isAsyncProcessMode) {
-            passed = passed && checkTimedResults(
-                expectedResults,
-                afterActionId,
-                TaskAction.process,
-                receivingOrg,
-                receivingOrgSvc,
-                maxPollSecs = 600
-            )
+        val processWaitTimeMillis = measureTimeMillis {
+            // if we are running in async mode, verify the correct number of 'process' records have been generated
+            if (isAsyncProcessMode) {
+                passed = passed && checkTimedResults(
+                    expectedResults,
+                    afterActionId,
+                    TaskAction.process,
+                    receivingOrg,
+                    receivingOrgSvc,
+                    maxPollSecs = 600
+                )
+            }
+        }
+
+        // Since processing starts as soon as we start submitting, the submission time plus
```

Comment:
Starting here and below is some kinda gloppy code that prints out useful statistics on the 'process' step.